sirepo/template/madx.py

```diff
@@ -328,12 +328,12 @@ def get_data_file(run_dir, model, frame, options=None, **kwargs):
 
 def import_file(req, **kwargs):
     text = pkcompat.from_bytes(req.file_stream.read())
-    assert re.search(r'\.madx$', req.filename, re.IGNORECASE), \
+    assert re.search(r'\.madx$|\.seq$', req.filename, re.IGNORECASE), \
```

Comment:
We've later learned that "assert" is ignored when python is run in optimized mode, so whenever you modify an assert() line, change it to a "raise AssertionError()"